src/Components/LoginForm/LoginForm.js

```diff
@@ -1,9 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './LoginForm.scss';
+import banner from '../../images/banner.jpg'
+import { NavLink } from 'react-router-dom';
 
 const LoginForm = () => {
-  return(
-    <p>LoginForm</p>
+  const [emailInput, handleEmail] = useState('');
```

Comment:
Same comment as before, I think we can refactor into one state object and use e.target.value to overwrite on the onChange

Reply:
We can format state in which ever way is easiest for us to send to the back end when we send it to the provider.